sbin/prepareWorkspace.sh

```diff
@@ -165,7 +167,20 @@ checkoutRequiredCodeToBuild() {
           echo "Failed cmd: git checkout \"${tag}\""
         fi
       else
-        echo "Failed cmd: git fetch origin \"refs/tags/${tag}:refs/tags/${tag}\""
+        echo "Checking out commit id ${tag}"
```

Comment:
Perhaps change this echo to "Unable to checkout as refs/tags/${tag}, attempting to checkout as commit id $tag"